src/App.js

```diff
@@ -1,7 +1,27 @@
-import React from 'react'
+import React, { useState} from 'react';
+import TodoList from './TodoList';
+import AddTodoForm from './AddTodoForm';
 
-export default function App() {
+
+
+
+function App() {
+  const [newTodo, setNewTodo] = useState(''); 
+  const [todoList, setTodoList] = useState([]);
+  const addTodo = (newTodo) =>{
+    setTodoList([...todoList ,newTodo])
```

Comment:
I did the same thing here!! haha This will work most of the time but react suggest us to pass a function to the setter state function to get the most recent state, if we plan on using it.

Like for instance,
```suggestion
    setTodoList((previousTodoList) => [...previousTodoList, newTodo])
```
So we used the old state to populate the new state with the older items then added our `newTodo`. Ahh React is fun isn't it 😆 Learning new frameworks "gotchas" are annoying but save you headaches when trying to figure out weird bugs later on.

Keep up the good work man 😄

Reply:
Try to fix this during the next exercise too 👍

Reply:
oh, thank you .Ive fixed it!

Reply:
Nice!! 💯